nestkernel/connection_manager.cpp

```diff
@@ -1288,33 +1315,35 @@ nest::ConnectionManager::get_sources( std::vector< index > targets,
 void
 nest::ConnectionManager::get_targets( std::vector< index > sources,
   std::vector< std::vector< index > >& targets,
-  index synapse_model )
+  index synapse_model,
+  std::string post_synaptic_element )
 {
-  thread thread_id;
+  unsigned int thread_id;
   std::vector< index >::iterator source_it;
   std::vector< std::vector< index > >::iterator target_it;
   targets.resize( sources.size() );
   for ( std::vector< std::vector< index > >::iterator i = targets.begin();
         i != targets.end();
-        i++ )
+        ++i )
   {
     ( *i ).clear();
   }
```

Comment:
I think the entire code from `targets.resize()` can be simplified. If I understand the code right, in the end `targets` should be a vector of `sources.size()` empty vectors. I think the idiomatic way to achieve this is
```c++
std::vector< std::vector< index > >( sources.size() ).swap( targets );
```